<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSensorTypeDialog</class>
 <widget class="QDialog" name="NewSensorTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить тип датчика</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-size: 9pt;
}

QPushButton
{
	background-color: rgb(218, 218, 218);
	border: none;
	border-radius: 6px;
	min-width: 80px;
	min-height: 30px;
	font-size: 8pt;
}

QPushButton:hover
{
	border-top: 1px solid black;
	border-left: 1px solid black;
	border-right: 2px outset black;
	border-bottom: 2px outset black;
}

QPushButton:pressed
{
	border: 2px inset black;
}

QLineEdit
{
	border: 1px solid black;
	border-radius: 6px;
}

QDoubleSpinBox
{
	border-radius: 6px;
	border: 1px solid gray;
	min-height: 20px;
	min-width: 60px;	
	max-width: 100px;	
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_49">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Наименование</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QLabel" name="label_50">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Штатные значения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dsbMin">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>65535.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_51">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dsbMax">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>65535.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSensorTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSensorTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
